scripts/govDocES.py

```diff
@@ -27,20 +28,22 @@ def main():
 
     dbManager.createSession()
 
-    for work in dbManager.session.query(Work) \
+    batchSize = 1000
+    for work in dbManager.session.query(Work.uuid, array_agg(Edition.measurements)) \
         .join(Edition) \
         .filter(Edition.measurements != None) \
         .filter(Edition.measurements != []) \
-        .filter(Edition.measurements != [{}]).all():
+        .filter(Edition.measurements != [{}]) \
+        .group_by(Work.uuid) \
+        .yield_per(batchSize):
 
             break_out_flag = False
-            
-            for edition in work.editions:
-                for measurement in edition.measurements:
+            for editionMeasurements in work[0]:
+                for measurement in editionMeasurements:
```

Comment:
Why does work[0] equal to the editionMeasurements and not the uuid or any other attribute?

Reply:
When you change the query from just `Work` to `Work.uuid, ...` it changes the behavior. Instead of getting an ORM object you get a tuple of the fields you requested. I can update this to make that clearer (also this should be `[1]` here!

Reply:
Appreciate the explanation.